src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -281,8 +290,8 @@ public Duration getAckWait() {
      * Gets the max delivery amount of this consumer configuration.
      * @return the max delivery amount.
      */
-    public long getMaxDeliver() {
-        return LongChangeHelper.MAX_DELIVER.getOrUnset(maxDeliver);
+    public int getMaxDeliver() {
```

Comment:
changed getters to return int since the underlying value is an int. This is not a breaking change since it is a primitive and used to be primitive long, so there would be no break if the user had read this value into an long variable.
applies to getMaxDeliver, getMaxAckPending, getMaxPullWaiting, getMaxBatch, getMaxBytes